go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/NVIDIA/go-nvml
 
 go 1.20
 
-require github.com/stretchr/testify v1.9.0
+require (
+	github.com/google/uuid v1.6.0
```

Comment:
As a matter of interest, where is this dependency from?

Reply:
It's to generate a UUID for the GPUs in the mock dgxa100 server

Reply:
Ah. OK. Thanks.